tests/Scaffolding/StaticSchemaTest.php

```diff
@@ -15,6 +15,12 @@
 
 class StaticSchemaTest extends SapphireTest
 {
+    protected function setUp()
```

Comment:
```suggestion
    protected function setUp(): void
```

This would be required for ss 5 compat